build_package_new.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2014-2021 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a new package build script which uses new package  build logic.
+# usage:
+#       build_package_new.py <name of the package> --output-dir <output directory>
```

Comment:
Would be good to add a quick check to this script which fails with more useful error message when running under Python version older than 3.8 - otherwise bit confusing error is printed and it's not immediately clear why it's failing to run.